vizro-core/tests/unit/vizro/models/_action/test_action.py

```diff
@@ -48,47 +102,30 @@ def test_create_action_mandatory_only(self, identity_action_function):
         assert action.inputs == []
         assert action.outputs == []
 
-    def test_create_action_mandatory_and_optional(self, identity_action_function):
-        function = identity_action_function()
-        inputs = ["component_1.property_A", "component_1.property_B"]
-        outputs = ["component_2.property_A", "component_2.property_B"]
-
-        action = Action(function=function, inputs=inputs, outputs=outputs)
-
-        assert hasattr(action, "id")
-        assert action.function is function
-        assert action.inputs == inputs
-        assert action.outputs == outputs
-
-    def test_is_model_inheritable(self, identity_action_function):
-        class MyAction(vm.Action):
-            pass
-
-        function = identity_action_function()
-        my_action = MyAction(function=function)
-
-        assert hasattr(my_action, "id")
-        assert my_action.function is function
-        assert my_action.inputs == []
-        assert my_action.outputs == []
+        # TODO: Should we test private attributes?
```

Comment:
In this case yes, we should. In fact we should also test `_parameters` and `_runtime_args` unless there's a good reason not to?